src/components/SmallDirectoryBlock.tsx

```
@@ -1,173 +1,160 @@
+          value={searchQuery}
+          onChange={(e) => setSearchQuery(e.currentTarget.value)}
+        />
+        <RadioGroup
+          value={roleFilter}
+          onChange={(value) =>
+            setRoleFilter(value as "CAMPER" | "STAFF" | "ADMIN")
+          }
+        >
+          <div className="flex gap-md mt-md">
+            {attendeeRoles.map((attendeeRole) => (
+              <Radio value={attendeeRole} label={getPluralRole(attendeeRole)} />
```

ESLint

[error] Missing "key" prop for element in iterator

(react/jsx-key)